src/map/MarkerBounds.stories.tsx

```diff
@@ -64,3 +65,27 @@ export const MarkerBounds = () => {
       />
   );
 };
+
+export const DoubleClick = () => {
```

Comment:
Is this story any different from the first?

Reply:
I meant to delete this 😆